src/kactl/graph/DFSMatching.h

```diff
@@ -1,4 +1,5 @@
 /**
+ * Name: Slow maximum matching (DFS)
```

Comment:
How about `$O(EV)$ maximum matching`? Unless it's a problem to add the $s.

Reply:
It shouldn't be a problem after merging #60.